test/test_tensor.py

```diff
@@ -64,6 +64,15 @@ def test_pytorch():
     for x,y in zip(test_tinygrad(), test_pytorch()):
       np.testing.assert_allclose(x, y, atol=1e-5)
 
+  @unittest.skipUnless(not DEFAULT_DEVICE, "float64 not supported on GPU")
```

Comment:
You need float64 for this?

Reply:
the test fails on GPU and I'm not really sure why.. below is the error message
```
self = <test.test_tensor.TestTinygrad testMethod=test_dropout>

    def test_dropout(self):
      Tensor.training = True
      n, rate = 1_000_000, 0.1
      w = Tensor.ones(n).dropout(rate)
      non_zeros = np.count_nonzero(w.data)
      expected = n * (1 - rate)
>     np.testing.assert_allclose(non_zeros, expected, rtol=1e-3)
E     AssertionError: 
E     Not equal to tolerance rtol=0.001, atol=0
E     
E     Mismatched elements: 1 / 1 (100%)
E     Max absolute difference: 899999.
E     Max relative difference: 0.99999889
E      x: array(1)
E      y: array(900000.)

test/test_tensor.py:73: AssertionError
```